axelor-crm/src/main/resources/views/Partner.xml

```diff
@@ -23,7 +23,8 @@
 	        <menu title="Tools" icon="fa-wrench" showTitle="true">
 	            <item title="Schedule Event" action="save,action-partner-view-schedule-event" readonlyIf="id == null" if="__config__.app.isApp('crm')"/>
 	            <item title="Create opportunity" action="save,action-partner-view-create-opportunity" readonlyIf="id == null" if="__config__.app.isApp('crm')"/>
-	            <item title="Create order" action="save,action-partner-view-create-order" readonlyIf="id == null || id &lt; 1" if-module="axelor-sale" if="__config__.app.isApp('sale')" showIf="isProspect || isCustomer"/>
+	            <item title="Create sale quotation" action="save,action-partner-view-create-order" readonlyIf="id == null || id &lt; 1" if-module="axelor-sale" if="__config__.app.isApp('sale')" showIf="isProspect || isCustomer"/>
+	            <item title="Create purchase quotation" action="save,action-partner-view-create-purchase-quotation" readonlyIf="id == null || id &lt; 1" if-module="axelor-purchase" if="__config__.app.isApp('purchase')" showIf="isProspect || isSupplier"/>
```

Comment:
Check only isSupplier not isProspect, prospect become customer only not supplier